test/uk/gov/hmrc/tai/service/journeyCache/JourneyCacheServiceSpec.scala

```diff
@@ -343,8 +342,8 @@ class JourneyCacheServiceSpec extends BaseSpec with BeforeAndAfterEach {
     "remove the cache" in {
       val sut = createSut
       when(journeyCacheConnector.flushWithEmpId(Matchers.eq(sut.journeyName), Matchers.eq(1))(any()))
-        .thenReturn(Future.successful(TaiSuccessResponse))
-      Await.result(sut.flushWithEmpId(1), 5 seconds) mustBe TaiSuccessResponse
+        .thenReturn(Future.successful(Done))
+      Await.result(sut.flushWithEmpId(1), 5 seconds) mustBe Done
```

Comment:
is this 5 seconds a limit on the response?

Reply:
This is just for testing to go from future to blocking code (if a future is called future a blocking is called present?) 
Sometimes this sort of code can cause timeouts whenever CI is busy...